internal/database/online/redis/util.go

```diff
@@ -6,8 +6,18 @@ import (
 	"github.com/oom-ai/oomstore/internal/database/online/kvutil"
 )
 
-func serializeRedisKey(revisionID int, entityKey interface{}) (string, error) {
-	prefix, err := kvutil.SerializeByValue(revisionID)
+func batchSerializeRedisKey(revisionID int, entityKey interface{}) (string, error) {
```

Comment:
maybe `serializeRedisKeyForBatch`?

Reply:
Sounds good! `serializeRedisKeyForBatchFeature` Maybe better? Because we know exactly what `serializeRedisKeyForBatch` means, but the others don't necessarily, what do you think?

Reply:
sounds good